API_design.md

```diff
@@ -0,0 +1,256 @@
+ # Restful API
+
+## Header
+
+### login_header
+```
+Host: api.orange.com
+Authorization: Bearer ACCESS_TOKEN
+Content-Type: application/json
+Accept: application/json
+User-Agent: OrangeApp/1.0
+
+```
+### guest_header
+`Allow anonymous access without requiring authentication headers`
+```
+
+Content-Type: application/json
+Accept: application/json
+User-Agent: OrangeApp/1.0
+
+```
+
+### Methods：GET，POST
+`https://api.orange.com/v1/products`
```

Comment:
The protocol and domain are not required for API Doc. The specific API URL is enough.

```suggestion
`/v1/products`
```